tools/clang/test/HLSLFileCheck/d3dreflect/rdat_mintarget/sm68_node_deriv.hlsl

```diff
@@ -0,0 +1,222 @@
+// RUN: %dxilver 1.8 | %dxc -T lib_6_8 %s | %D3DReflect %s | %FileCheck %s -check-prefixes=RDAT
+
+// Ensure that categories of deriv ops are allowed for node shaders.
```

Comment:
As with the previous, this at least seems to check that and that the OptFeatureInfo_UsesDerivatives  flag gets set. 
```suggestion
// Ensure that the OptFeatureInfo_UsesDerivatives flag gets set
// for different categories of deriv ops in node shaders.
```

Reply:
I can add that it checks that the `OptFeatureInfo_UsesDerivatives` flag gets set as well, but this is also to explicitly regression test that node shaders are considered compatible with different categories of derivative ops, since that was another bug fixed (some deriv op categories didn't say that they were supported in node shaders, see update in hctdb.py).